DynamicLanguageRuntime/Remotion.ThirdPartyDependency.DynamicLanguageRuntime.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\_._">
+      <Pack>true</Pack>
+      <PackagePath>\</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <Owner>RUBICON IT GmbH</Owner>
+    <Version>1.1.0.10</Version>
+    <Title>Microsoft Dynamic Language Runtime</Title>
+    <Description>The Dynamic Language Runtime enables language developers to more easily create dynamic languages for the .NET platform. In addition to being a pluggable back-end for dynamic language compilers, the DLR provides language interop for dynamic operations on objects. The DLR has common hosting APIs for using dynamic languages as libraries or for scripting in your .NET applications.</Description>
```

Comment:
I'm missing the source information from the package.ps1 file in re-motion. We should do the following:
1) add this as a Notice.txt into the nuget package and/or in the description field. Not 100% sure if it's needed in the description. easier is the notice.txt, I guess.
2) add a link to the relevant github repo and commithash